Utilities/Get-SubscriptionUsage.ps1

```diff
@@ -236,6 +239,11 @@ foreach ($region in $allRegions)
             $currentPublicIPs += 1
         }
     }
+    try {
+        $RegionUsagePercent = $([math]::Round($currentUsedCores*100/$currentRegionAllowedCores,1))
```

Comment:
this is not the way you should do it, you are still dividing by zero. use an if instead.

Reply:
@replaced with if.